typhon/suite.py

```diff
@@ -217,7 +217,8 @@ def add_device(self, device, children=True, methods=None, image=None):
         super().add_device(device)
         # Add the device to the main panel
         self.device_panel.add_device(device, methods=methods)
-        self.device_panel.add_image(image)
+        if image:
+            self.device_panel.add_image(image)
```

Comment:
What does this change do?

Reply:
This was a bug. If you called this in the past with `image=None` we would make a `PyDMDrawingImage` and set the `filename` to `None`.